From: =?utf-8?q?Alberts_Muktup=C4=81vels?= <alberts.muktupavels@gmail.com>
Date: Tue, 12 Feb 2019 03:00:21 +0200
Subject: core: when we uninstall a job, add unit to dbus queue

Commit e6d05912cb1785d8c75eb40545beb8a7c6753cb9 added unit to dbus
queue on job install. Do same on job uninstall to make sure we get
PropertiesChanged signal.

(cherry picked from commit 52c6c9eaecb493cc4d8a146bf67d93c8aea862c2)
---
 src/core/job.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/core/job.c b/src/core/job.c
index f635b7e..fc212d0 100644
--- a/src/core/job.c
+++ b/src/core/job.c
@@ -151,6 +151,8 @@ void job_uninstall(Job *j) {
 
         unit_add_to_gc_queue(j->unit);
 
+        unit_add_to_dbus_queue(j->unit); /* The Job property of the unit has changed now */
+
         hashmap_remove_value(j->manager->jobs, UINT32_TO_PTR(j->id), j);
         j->installed = false;
 }
